app/src/main/res/values/strings.xml

```diff
@@ -72,7 +72,7 @@
     <string name="main_not_numeric">%1$s is not a number.</string>
     <string name="main_invalide_trainerlvl">Invalid Trainer Level!</string>
     <string name="notification_title">GoIV Running - Level %1$s</string>
-    <string name="notification_text">Tap to open</string>
+    <string name="notification_open_app">Open app</string>
```

Comment:
I changed the string because now that this text is inside an action button it's obviuos it needs to be touched to start the action. Also, shorter text fits better inside the notification.
Also "click here to..." or "tap here to..." are not good UI/UX examples, actions should be self-explanatory through a clear design language. A notification action does the trick!